guides/plugins/plugins/storefront/add-custom-controller.md

```diff
@@ -19,7 +19,7 @@ Here's a video explaining the basics about storefront controllers from our free
 
 ### Storefront Controller class example
 
-First of all we have to create a new controller which extends from the `StorefrontController` class. A controller is also just a service which can be registered via the service container. Furthermore, we have to define our `RouteScope` via annotation, it is used to define which domain a route is part of and **needs to be set for every route**. In our case the scope is `storefront`.
+First of all we have to create a new controller which extends from the `StorefrontController` class. A controller is also just a service which can be registered via the service container. Furthermore, we have to define our `Route` with `defaults` and `_routeScope` via annotation, it is used to define which domain a route is part of and **needs to be set for every route**. In our case the scope is `storefront`.
 
```

Comment:
@shyim I think guides should represent current best practices. That's why think a simple hint about the old way of defining a route scope is enough.
```suggestion

{% hint style="info" %}
Prior to Shopware 6.4.11.0 the `_routeScope` was configured by a dedicated annotation: `@RouteScope`. This way of defining the route scope is deprecated for the 6.5 major version.
{% endhint %}

```